tests/testthat/test-aorsf_data.R

```diff
@@ -0,0 +1,176 @@
+library(testthat)
+
+mtcars_orsf <- mtcars
+mtcars_orsf$vs <- factor(mtcars_orsf$vs)
+mtcars_na <- mtcars_orsf
+mtcars_na$cyl[1] <- NA
+
+
+test_that("regression model object", {
+
+  skip_if_not_installed("aorsf", "0.1.3")
+
+  set.seed(1234)
+  aorsf_regr_fit <- aorsf::orsf(
+    # everyone's favorite
+    data = mtcars_orsf, formula = mpg ~ .,
+    # faster
+    n_tree = 10,
+    # requested default from tidymodels
+    n_thread = 1
+  )
+
+  # formula method
+  regr_spec <- rand_forest(trees = 10) %>%
+    set_engine("aorsf") %>%
+    set_mode("regression")
+
+  # not sure if the expect_error is needed, but copying from censored
+  set.seed(1234)
+  expect_error(
+    bonsai_regr_fit <- fit(regr_spec, data = mtcars_orsf, mpg ~ .),
+    NA
+  )
```

Comment:
```suggestion
  set.seed(1234)
  expect_no_condition(
    bonsai_regr_fit <- fit(regr_spec, data = mtcars_orsf, mpg ~ .)
  )
```

Thanks for the code comment here! `expect_error(..., NA)` is a funky way of testing that there's no error raised and is generally a pattern we're starting to transition away from (though have admittedly done so inconsistently so far). Could you adjust to this pattern in this PR?

Reply:
Ah that makes sense - will do!